config_models/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = '2.0.0'
+__version__ = '2.0.1'
```

Comment:
Looks like someone beat you to it.  You'll need to use 2.0.2 and you should update the CHANGELOG.rst.  Thanks.

Reply:
:ok_hand: